how-to/common/public/components/fin-toggle-class.js

```diff
@@ -0,0 +1,124 @@
+customElements.define(
+  "fin-toggle-class",
+  class ToggleClass extends HTMLElement {
+    constructor() {
+      super();
+    }
+
+    /**
+     * Runs each time the element is appended to or moved in the DOM
+     */
+    connectedCallback() {
+      this.sync();
+      this.applySettings(false);
+      this.addEventListener("click", this.onclick);
+    }
+
+    /**
+     * Runs when the element is removed from the DOM
+     */
+    disconnectedCallback() {
+      this.removeEventListener("click", this.onclick);
+    }
+
+    sync() {
+      if (window.fin !== undefined) {
+        this.key = "fin-toggle-class-" + fin.me.identity.uuid;
+
+        const sync = this.getAttribute("sync") || "true";
+        if (sync.toLowerCase() !== "false") {
```

Comment:
Instead of the double negative can we say `=== "true"`

Reply:
was just trying to ensure that the sync is applied even if someone enters the wrong value in the attribute (which is why it checks for false).